packages/orbit-design-tokens/package.json

```diff
@@ -43,6 +34,7 @@
     "lodash": "^4.17.14",
     "memfs": "^3.2.0",
     "optimist": "^0.6.1",
+    "style-dictionary": "^2.10.2",
     "theo": "^8.1.5",
```

Comment:
Also, we don't need `theo` anymore, right?
```suggestion
```

Reply:
Noted. Will be updated in upcoming PR 🙂